packages/fast-components-react-base/app/components/container.tsx

```diff
@@ -0,0 +1,16 @@
+
+import * as React from "react";
+
+export interface IContainerProps {
+    slot: string;
+}
+
+export default class Container extends React.Component<IContainerProps, {}> {
+    public render(): JSX.Element {
+        return (
+            <div>
```

Comment:
We need to pass slot somewhere, don't we?

Reply:
You do not need to pass it anywhere actually, since all of our getChildSlot methods do is look for the passed prop, not the prop on an element.